src/guide/essentials/watchers.md

```diff
@@ -244,6 +244,7 @@ export default {
 }
 ```
 
+In above scenario the initial execution of handler will happen after `beforeCreate` hook, but before `created`.
```

Comment:
Keeping in mind the outline presented in the [lifecycle diagram](https://vuejs.org/guide/essentials/lifecycle.html#lifecycle-diagram), I think it might be worth mentioning that the watcher is called after the various properties have been initialised, rather than mentioning `beforeCreate` directly. Maybe something like...

```suggestion
The initial execution of the handler function will happen just before the `created` hook. Vue will have already processed the `data`, `computed`, and `methods` options, so those properties will be available on the first invocation.
```

Reply:
Thanks, I like the suggested wording.